lang/cs/Org.Apache.REEF.Bridge.Core.Common/Driver/IDriverClientService.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+namespace Org.Apache.REEF.Bridge.Core.Common.Driver
+{
+    /// <summary>
+    /// The interface that initializes the driver client.
+    /// </summary>
+    internal interface IDriverClientService
+    {
+        /// <summary>
+        /// Start the driver client service.
+        /// </summary>
+        void Start();
+
+
+        /// <summary>
+        /// Await for termination.
+        /// </summary>
+        void AwaitTermination();
+    }
+}
```

Comment:
`***BOOKMARK*** 2018-08-08 16:00`
This is a bookmark for myself where to resume the code review. Please ignore :)